folium/folium.py

```diff
@@ -802,9 +802,9 @@ def geo_json(self, geo_path=None, geo_str=None, data_out='data.json',
             keyword argument will enable conversion to GeoJSON.
         reset: boolean, default False
             Remove all current geoJSON layers, start with new layer
-        freescale: if True use free format for the scale, where min and max values
-            are taken from the data. It also allow to plot allow to plot values < 0 
-            and float legend labels.
+        freescale: if True use free format for the scale, where min and max
+        values are taken from the data. It also allow to plot allow to plot
+        values < 0 and float legend labels.
 
```

Comment:
I guess there's a typo here, isn't it ?
 you may keep the indent, and avoid the duplication of `allow to plot`